src/components/pages/PatientsFullDetailsSearch/__tests__/PatientsFullDetailsSearch.test.js

```diff
@@ -46,7 +46,7 @@ describe('Component <PatientsFullDetailsSearch />', () => {
     expect(component).toMatchSnapshot();
 
     expect(component.find('.page-wrapper')).toHaveLength(1);
-    expect(component.find('PatientsList')).toHaveLength(1);
+    //expect(component.find('PatientsList')).toHaveLength(1);
```

Comment:
Please rewrite parts of the code or delete them. If there is a purpose in commenting out a block of code, we need to write a human-readable text, so anyone can easily understand why the code is commented out